packaging/bundle-validation/Dockerfile

```diff
@@ -27,3 +27,5 @@ RUN cp $DERBY_HOME/lib/derbyclient.jar $SPARK_HOME/jars/
 RUN cp conf/spark-defaults.conf $SPARK_HOME/conf/
 RUN if [[ $SPARK_HOME == *"spark-3.2"* ]] || [[ $SPARK_HOME == *"spark-3.3"* ]]; \
     then printf "\nspark.sql.catalog.spark_catalog org.apache.spark.sql.hudi.catalog.HoodieCatalog\n" >> $SPARK_HOME/conf/spark-defaults.conf; fi
+RUN printf "\ntaskmanager.numberOfTaskSlots: 2\n" >> $FLINK_HOME/conf/flink-conf.yaml
+RUN printf "\nlocalhost\n" >> $FLINK_HOME/conf/workers
```

Comment:
It means the validation only has 1 TM(task manager) with 2 slots, which should be okey for the test because the compaction task parallelisim expects to be 1, just to note that this could be risky that we may run out of slots.